src/components/container/SeverityCalendar/SeverityCalendar.tsx

```diff
@@ -145,6 +147,12 @@ export default function (props: SeverityCalendarProps) {
     if (!data) {
         return <LoadingIndicator innerRef={props.innerRef} />;
     } else {
-        return <Calendar innerRef={props.innerRef} className="mdhui-simple-calendar" year={intervalStart.getFullYear()} month={intervalStart.getMonth()} dayRenderer={renderDay} />;
+        return (<>
+            {dateRangeContext && dateRangeContext?.intervalType !== "Month" && 
+                <div className="mdhui-severity-calendar-date-interval-not-supported" ref={props.innerRef}>Severity calendar does not support week view at this time</div>}
```

Comment:
Same comment here.  I would move this check up to just after acquiring the `dateRangeContext`.  No need to load data or compute anything if it won't be able to render anyways.